openapi/paths/risk-scores.yaml

```
@@ -0,0 +1,62 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Risk Score
+  summary: Retrieve risk score rules
+  operationId: GetRiskScoreRules
+  x-sdk-operation-name: getAll
+  security:
+    - SecretApiKey: []
+    - JWT: []
+    - ApplicationJWT: []
+  description: Retrieve risk score rule.
```

nitpick: I think this should be plural
```suggestion
  description: Retrieves risk score rules.
```